e2e/operator_installation_test.go

```diff
@@ -0,0 +1,70 @@
+package e2e_test
+
+import (
+	. "github.com/maistra/istio-workspace/e2e/infra"
+	"github.com/maistra/istio-workspace/pkg/naming"
+	testshell "github.com/maistra/istio-workspace/test/shell"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"strings"
+	"time"
+)
+
+var _ = Describe("Operator Installation Tests", func() {
+
+	Context("local installation", func() {
+
+		It("install ike operator into specified namespace", func() {
+			// given
+			projectName := "ike-local-installation-defined-namespace-" + naming.RandName(16)
+			<-testshell.Execute(NewProjectCmd(projectName)).Done()
+			defer func() {
+				<-testshell.Execute(DeleteProjectCmd(projectName)).Done()
+			}()
+			PushOperatorImage(projectName)
```

Comment:
I think it's time we should optimize it and have a shared image built as the previous step of the pipeline. WDYT?

Reply:
```
build operator image
build test images

test
  tag/push operator image to ns
  tag/push test image to ns
```

?

Reply:
I was thinking about pushing to public repo so that we avoid test-level pushes too

Reply:
Yeah, I was about to comment if we should just push to quay. I'm good with that.

Reply:
Created #313